acceptance_tests/features/steps/view_response_status.py

```diff
@@ -0,0 +1,33 @@
+from behave import when, given, then
+
+from acceptance_tests import browser
+from acceptance_tests.features.pages import reporting_unit
+
+RESPONSE_STATUS = browser.find_by_name('tbl-ce-status')
+
+
+@then('the "{survey}" "{period}" collection exercise for ru "{ru_ref}" displays a view link')
+def assert_view_response_status_link(_, survey, period, ru_ref):
+    reporting_unit.go_to(ru_ref)
+    reporting_unit.click_data_panel('Bricks')
+    view_link = '<a href="/case/49900000001/response-status?survey=Bricks&amp;period=202001">View</a>'
```

Comment:
should you be hard-coding the survey, period and RU ref in this link?

Reply:
So for this instance its ok, cos during the setup they are explicitly set

Reply:
why pass the variables in if you're going to hard code it?

Reply:
It's a bit misleading that the method has been set up to take any survey, period and RU but it'll fail the assertion if you give it anything other than 'Bricks', 49900000001' and '202001'.